src/HttpEnvelopingRequest.ts

```diff
@@ -0,0 +1,38 @@
+import type { Either } from '@rsksmart/rif-relay-client/dist/common/utility.types';
```

Comment:
I would also think about moving this class to the definitions folder

Reply:
Done